Makefile

```diff
@@ -96,7 +96,6 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
 	pip-compile --upgrade -o requirements/production.txt requirements/production.in
 	# Let tox control the Django version for tests
-	grep -e "^django==" requirements/production.txt > requirements/django.txt
```

Comment:
why remove django.txt

Reply:
We were using it exclusively for django1.11. Since we have removed django1.11, I think we don't need it now.

Reply:
We're going to need it again as soon as we start testing with Django 3.0 (which will be pretty soon).